Server/mods/deathmatch/logic/CAccount.cpp

```diff
@@ -289,6 +289,15 @@ bool CAccount::RemoveSerial(const SString& strSerial)
     return false;
 }
 
+//
+// Replace the serial number for a specific account
+//
+void CAccount::SetAccountSerial(const std::string strSerial)
+{
+    m_strSerial = strSerial;
+    m_pManager->MarkAsChanged(this);
```

Comment:
You aren't checking if the serial is even valid. That will cause potential security issues in the future

Reply:
adjusted be766cf